### DISCUSSION TITLE
Telegram

### DISCUSSION THREAD

**Discussion by giscus** [AwaitQuality/windowsonarm#15] [General]
# Telegram

Check if Telegram is ARM ready on Windows and can be used on Windows ARM devices such as the Surface with the Snapdragon X Elite.

https://windowsonarm.org/535458d1-33ec-4848-a4a4-eba060c2d115

**Comment by LlamaNL** [comment]
There is no ARM64 Telegram Desktop client anywhere on the Telegram site

**Reply by OpenSource03** [reply]
Thank you for reporting, I've corrected that :)

**Comment by OpenSource03** [comment]
Further info available at
https://github.com/telegramdesktop/tdesktop/issues/25266

**Comment by timetrvlr** [comment]
There is now an arm native version of telegram!

https://www.windowscentral.com/software-apps/telegram-launches-native-windows-on-arm-support
